pages/glossary/keyboard-actionable-element.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Keyboard Actionable Element
+key: keyboard-actionable-element
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An HTML element is _keyboard actionable_ if all the following are true:
+
+- the element is part of [sequential focus navigation][]; and
+- the element is [visible][] when [focused][]; and
+- it is possible to [fire a click event][] at the element by use of keyboard.
```

Comment:
Under what circumstance would this not be true? This one is a little tricky. What do you mean with "by use of the keyboard"? If I have an on-screen cursor that I control using arrow keys, and use space to fire a click event from the cursor, does that count? Arguably it would, but I suspect that's not what you mean. Additionally, are you expecting the click event to be fired when the element is focused? Could event bubbling be involved?